pkg/sippyserver/server.go

```diff
@@ -99,11 +100,20 @@ func (s *Server) printHtmlReport(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	var numDays int
+	t := req.URL.Query().Get("endDay")
```

Comment:
doesn't this get set on the "detail" page today?

Reply:
yes, but the `detailed` page has a ton of options and isn't really pre-computable (we could probably cache things after they are computed the first time but that's a little bit more involved I think). This just adds the parameter to the regular page